### ISSUE TITLE
I saw complete details, there would be lots of work to be done. I can contribute building backend and it's api's for web and mobile

### ISSUE THREAD

**Issue by mehfoozdev** [codervivek5/VigyBag#983]
You can assign me any backend and api task. Thanks

**Comment by codervivek5** [comment]
Let's explain first what you want to add and make change so that I can manage a mentor for you so that they can review your code and merge your PR

**Comment by mehfoozdev** [comment]
Yes, first of all I saw there is no api for products. In product app there is no serializers, I will start working from designing products api (serialize and deserialize) so we can use it for both web and mobile smoothly.  We can work with Api instead of directly rendering data from database like in products, payment and order status, when there is lots of data, it will take time. Implementation with api will be useful as you are working with both web and mobile app. 

1) Products API
2) Contact API 

after these two i can work with order status too, when user purchase something we can handle and manage their order status.

3) Order management
4) python django code optimization

**Comment by mehfoozdev** [comment]
First you can assign me products and contact api. I can manage all the backend stuff for these two api.

**Comment by codervivek5** [comment]
@mehfoozdev ok got it ... start working on it and kindly share you update day by day ... means how much work have you done

**Comment by mehfoozdev** [comment]
okay

**Comment by mehfoozdev** [comment]
I have setup project, will work on it. 
![image](https://github.com/codervivek5/VigyBag/assets/168359047/38894c6d-00e2-43e4-a941-bad94087c4b2)

**Comment by codervivek5** [comment]
let's run the backend app and let me know is it working properly or not if yes then contact me as soon as possible

**Comment by mehfoozdev** [comment]
Hey, Let me know from where i can find .env file? or provide me secrete key. Rest of the things I have setup locally. 
![image](https://github.com/codervivek5/VigyBag/assets/168359047/d2ac9104-045e-4375-a3d0-7e0e8d0e60fa)

**Comment by Shubham-2004** [comment]
You can assign me any backend and rest api task.

**Comment by codervivek5** [comment]
check it again I have fix it ... secreat key issue

**Comment by mehfoozdev** [comment]
yes it's working

**Comment by mehfoozdev** [comment]
![image](https://github.com/codervivek5/VigyBag/assets/168359047/6489ec2d-f196-417d-85b0-ada42129dd04)
Giving here this error

**Comment by mehfoozdev** [comment]
implemented nested serializer of products, working on views logic.

**Comment by codervivek5** [comment]
ok Thankyou for the updates.

**Comment by codervivek5** [comment]
@Ansh101112  ok I will assign to you

**Comment by dharapandya85** [comment]
Issue available? Please assign me this issue.

**Comment by dharapandya85** [comment]
@codervivek5 , why the root directory has `routes` folder, which has product routes file`(productRoutes.js)`. I created the same product routes component within backend folder, with GET, PUT, UPDATE products API calls, but the frontend is not interacting with backend. When adding or deleting a product, there is not API call at backend. Can you explain me? 
The expected behavior at backend must be `/api/get-product` or `/api/update-product`.

**Comment by codervivek5** [comment]
> [@codervivek5](https://github.com/codervivek5) , why the root directory has `routes` folder, which has product routes file`(productRoutes.js)`. I created the same product routes component within backend folder, with GET, PUT, UPDATE products API calls, but the frontend is not interacting with backend. When adding or deleting a product, there is not API call at backend. Can you explain me? The expected behavior at backend must be `/api/get-product` or `/api/update-product`.

Actually for this project we have a seperate backend ... as you can see this is for the contributors only ... if you need then I'll share the backend repo with you then you can see only then you can make required changes

**Comment by dharapandya85** [comment]
@codervivek5, it's confusing that after adding proper routes, no API calls on backend. Do share me the backend.
